File: ballast-client/npm/src/signalr/signalr-client-chat-service.ts
import { IChatService, IChatMessage } from "ballast-core";

export class SignalRClientChatService implements IChatService {
    dispose(): void {
        throw new Error("Method not implemented.");
    }
    sendMessageAsync(message: IChatMessage): Promise<void> {
        throw new Error("Method not implemented.");
    }    
}

File: ballast-client/dotnet/src/signalr/SignalRClientChatService.cs
using System.Threading.Tasks;
using Ballast.Core.Application.Models;
using Ballast.Core.Application.Services;

namespace Ballast.Client.SignalR
{
    public class SignalRClientChatService : IChatService
    {
        public void Dispose()
        {
            throw new System.NotImplementedException();
        }

        public Task SendMessageAsync(ChatMessage message)
        {
            throw new System.NotImplementedException();
        }
    }
}